app/controllers/samples_controller.rb

```diff
@@ -54,6 +54,7 @@ class SamplesController < ApplicationController
   PAGE_SIZE = 30
   MAX_PAGE_SIZE_V2 = 100
   MAX_BINS = 34
+  MIN_CLI_VERSION = '0.5.0'.freeze
```

Comment:
We should force it to be at least 0.7.3 which fixes the long files bug.

Reply:
👍 changed to 0.7.3.